_data/sites.json

```diff
@@ -19471,6 +19471,17 @@
         ]
     },
 
+    {
+        "name": "Skool",
+        "url": "https://www.skool.com/privacy?rt=udt",
+        "difficulty": "medium",
+        "notes": "Use this [link](https://www.skool.com/privacy?rt=udt) and submit a request to delete your account permanently. You will receive a confirmation mail shortly. Note that this will not cancel payments processed outside of the platform. Best to contact the group admins for further assistance.",
```

Comment:
```suggestion
        "notes": "Use the link to submit a request to delete your account permanently. You will receive a confirmation mail shortly. Note that this will not cancel payments processed outside of the platform. Best to contact the group admins for further assistance.",
```